## Diagnostics

### Abstract

### Introduction

Identifying individuals who have contracted COVID-19 is crucial to slowing down the global pandemic.
Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea's successful strategy for controlling the spread of the virus.
Following the first release of the genetic sequence of the virus by Chinese officials on January 10, 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Diagnostic approaches utilizing a variety of methods are currently or have been developed.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Molecular tests are essential for identifying individuals for treatment and alerting their contacts to quarantine and be alert for possible symptoms.
While serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past, their potential for false positives means that they are not currently recommended for this use.
However, serological tests provide population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
Thus, they may be useful in efforts to better understand the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.

Understanding the fundamental organization of the human immune response to viral threats is critical to understanding the varied response to SARS-CoV-2.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.

In the case of COVID-19, there is also concern about the immune system becoming over-active.
One of the main immune responses contributing to the onset of acute respiratory distress syndrome (ARDS) in COVID-19 patients is cytokine storm syndrome (CSS), which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
In addition to respiratory distress, this mechanism can lead to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].

### Molecular Tests

Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample and thus to diagnose an active viral infection.
This first requires identifying which biospecimens are likely to contain the virus during infection and then acquiring these samples from the patient(s) to be tested.
Common sampling sources for molecular tests include nasopharyngeal cavity samples, such as throat wash and saliva [@doi:10/ggp4qx], and stool samples [@doi:10.1002/jmv.25742].
Once a sample is acquired from a patient, molecular testing will utilize a number of steps, described below, to analyze a sample and identify whether evidence of SARS-CoV-2 is present.
When testing for RNA viruses like SARS-CoV-2, pre-processing is necessary in order to create DNA from the RNA sample.
The DNA can then be amplified with PCR.
Some tests use the results of the PCR itself to determine whether the pathogen is present, but in other cases, it may be necessary to sequence the amplified DNA.
Sequencing requires an additional pre-processing step: library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters [@doi:10.1016/j.biotechadv.2020.107537].
In some cases, library preparation can involve other modifications of the sample, such as adding "barcoding" to identify a particular sample in the sequence data, which is useful for pooling samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
Sequential pattern matching is then used to identify unique subsequences of the virus that identify it in specific.
If sufficient subsequences are found, the test is considered positive.

#### RT-PCR

Real-Time Polymerase Chain Reaction (RT-PCR) tests determine whether a target is present by measuring the rate of amplification during PCR compared to a standard.
When the target is RNA, such as in the case of RNA viruses, the RNA must be converted into complementary DNA during pre-processing.
The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
This test uses RT-PCR with reverse transcription [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the _ORF1b_ of the RNA-dependent RNA polymerase (RdRP), the Envelope protein gene (_E_), and the Nucleocapsid protein gene (_N_).
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this test utilizes two probes against RdRP, one of which is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Importantly, this assay was not found to return false positive results.

#### qRT-PCR

Chinese researchers developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two gene regions of the viral genome, _ORF1b_ and _N_ [@doi:10.1093/clinchem/hvaa029].
This assay was tested on samples coming two COVID-19 patients and a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the _N_ gene to screen patients, while the _ORF1b_ gene region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that SARS-CoV-1 and SARS-CoV-2 are the only sarbecoviruses currently known to infect humans, a positive test can be assumed to indicate that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.

##### dPCR

Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional real-time quantitative PCR.
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for much higher throughput.
While dPCR equipment is not yet as common as that for RT-PCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, comprised by 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using RT-PCR.
Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
All of these individuals tested positive using ddPCR.
Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR.
Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR.
While this study did not provide a complete head-to-head comparison to RT-PCR in all aspects, e.g., no samples testing positive using RT-PCR were evaluated by ddPCR, the study shows the potential of dPCR for viral detection even in highly diluted samples.
In a second study, Dong et al. [@doi:10.1101/2020.03.14.20036129] compared the results of qRT-PCR and ddPCR testing for SARS-CoV-2 in 194 samples, including 103 samples from suspected patients, 75 from contacts and close contacts, and 16 from suspected convalescents.
Of the 103 suspected patient samples, 29 were reported as positive, 25 as negative, and 49 as suspected by qRT-PCR; all patients were later confirmed to be SARS-CoV-2 positive.
Of the qRT-PCR negative or suspected samples, a total of 61 (17 negative and 44 suspected) were later confirmed to be positive by ddPCR, improving the overall detection rate among these patients from 28.2% to 87.4%.
Of 75 patient samples from contacts and close contacts, 48 tested negative with both methods, and these patients were observed to remain healthy.
Within the remaining 27 patient samples, 10 tested positive, 1 negative, and 16 suspect with qRT-PCR.
Fifteen out of 16 suspect samples and the negative test results were overturned by ddPCR, decreasing the rate of suspect cases from 21% to 1%.
Importantly, all samples that tested positive using qRT-PCR also tested positive using ddPCR.
Among the 16 convalescent patients, qRT-PCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Overall, these studies suggest that dPCR is a promising tool for overcoming the problem of false-negative SARS-CoV-2 testing.

#### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. [@doi:10.1101/2020.03.26.20039438], who found they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.

##### CRISPR-based Detection

Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection.
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations [@doi:10.1126/science.aam9321].
The target RNA is amplified by RT-RPA and T7 transcription, and the amplified product activates Cas13a.
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA.
An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs [@doi:10.1371/journal.ppat.1008705].
It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and nine were ruled out by metagenomic next-generation sequencing, 17 nCoV-/HCoV+ cases and 36 samples from healthy subjects), and a reaction turnaround time of 40 minutes.
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].
Lateral flow test strips are simple to use and read, but there are limitations in terms of availability and cost per test.
Another group therefore proposed the CREST protocol (Cas13-based, Rugged, Equitable, Scalable Testing), which uses a P51 cardboard fluorescence visualizer, powered by a 9V battery, for the detection of Cas13 activity instead of immunochromatography [@doi:10.1128/JCM.02402-20].
CREST can be run, from RNA sample to result, with no need for AC power or a dedicated facility, with minimal handling in approximately 2 hours.
Testing was performed on 14 nasopharyngeal swabs.
CREST picked up the same positives as the CDC-recommended TaqMan assay with the exception of one borderline sample that displayed low-quality RNA.

The DETECTR method (DNA Endonuclease-Targeted CRISPR Trans Reporter) from Mammoth Biosciences involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, which is a rapid, isothermal nucleic acid amplification technique, and application of their Cas12-based technology.
In their assay, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions [@doi:10.1038/s41587-020-0513-4].
In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the E and N genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay.
The estimated limit of detection was 10 copies per &mu;l reaction, versus 1 copy per &mu;l reaction for the CDC assay.
These results have been confirmed by other DETECTR approaches.
Using RTRPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per &mu;l of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
Using a similar approach, another group reported detection at 1 copy per &mu;l [@doi:10.1371/journal.pbio.3000978].
The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual crRNAs (which increases sensitivity).
This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1038/s41467-020-18575-6].
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1038/s41421-020-0174-y].
It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1073/pnas.2010254117].
The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay to achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.

All these methods required upstream nucleic acid amplification prior to CRISPR-based detection. 
They are relying on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
In contrast, type III CRISPR systems have the unique property of initiating a signalling cascade, which could boost the sensitivity of direct RNA detection. 
A study tested this hypothesis using the type III-A CRISPR RNA (crRNA)-guided surveillance complex from Thermus thermophilus [@doi:10.1101/2020.10.14.20212670]. 
They showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, pyrophosphates) that can all be used to detect SARS-CoV-2 RNA. 
Direct detection was 100-fold more sensitive than direct detection by Cas13. 
Detection of viral RNA in patient samples still required an initial nucleic acid amplification step, but improvements may in the future remove that requirement.

This goal of amplification-free detection was later achieved for a Cas13a-based system [@doi:10.1016/j.cell.2020.12.001]. 
This approach combined multiple CRISPR RNAs to increase Cas13a activation, which is detected by a fluorescent reporter. 
Importantly, because the viral RNA is detected directly, the test yields a quantitative measurement rather than a binary result. 
The study also shows that fluorescence can be measured in a custom-made dark box with a mobile phone camera and a low-cost laser illumination and collection optics. 
This makes this approach a truly portable assay for point-of-care diagnostics. 
The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n=20).

There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
In the largest study to date, DETECTR was compared to qRT-PCR on 378 patient samples [@doi:10.1093/infdis/jiaa641].
The authors reported a 95% reproducibility.
Both techniques were equally sensitive in detecting SARS-CoV-2.
Lateral flow strips showed a 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
A method based on a Cas9 ortholog from Francisella novicida (FnCas9) achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1101/2020.09.13.20193581].

#### Limitations of Molecular Tests

Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:

1. False-negative responses, which can present a significant problem to large-scale testing.
   To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
2. Uncertainty surrounding the SARS-CoV-2 viral shedding kinetics, which could affect the result of a test depending on when it was taken [@doi:10.1038/d41587-020-00002-2].
3. Type of specimen, as it is not clear which clinical samples are best for detecting the virus [@doi:10.1038/d41587-020-00002-2].
4. Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities [@doi:10.1126/science.abb8400].
5. Timing of the test, which might take up to 4 days to return results [@doi:10.1126/science.abb8400].
6. The availability of supplies for testing, including swabs and testing media, has been limited [@doi:10.1128/JCM.00512-20].
7. Because the guide RNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.

Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome.
As noted above, false negatives are a significant concern for several reason.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].

### Serological Tests

Although diagnostic tests based on the detection of genetic material can be quite sensitive, they cannot provide information about the extent of the disease over time.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, serological tests, which use serum to test for the presence of antibodies against SARS-CoV-2, are significantly more informative.
Additionally, serological tests can help scientists to understand why the disease has a different course among patients, as well as which strategies might work to manage the spread of the infection.
Furthermore, serological tests hold significant interest because of the possibility that they could provide information relevant to advancing economic recovery and allowing reopenings.
For instance, people that had developed antibodies might be able to return to work, assuming (still-unproven) protective immunity [@doi:10.1101/2020.04.14.20065771].
Some infectious agents can be controlled through "herd immunity", which is when a critical mass within the population acquires immunity through vaccination and/or infection, preventing an infectious agent from spreading widely.
A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R0)) fraction of the population must be immune or, equivalently, less than (1/R0) fraction of the population susceptible [@isbn:9780199209996].
However, for SARS-CoV-2 and COVID-19, the R0 and mortality rates that have been observed suggest that relying on herd immunity without some combination of vaccines, proven treatment options, and strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.

#### Sustained Immunity to COVID-19

In the process of mounting a response to a pathogen, the immune system produces antibodies specific to the pathogen.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
The two immunoglobulin classes most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
Following SARS infection, IgM and IgG antibodies were detected in the second week post-infection.
IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180.
IgG titers peaked by day 60, and persisted in all donors through the two-year duration of study [@doi:10.1111/j.1440-1843.2006.00783.x].
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
These results suggest that immunity to SARS-CoV-1 is sustained for at least a year.

The persistence of antibodies to SARS-CoV-2 remains under investigation.
Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year [@doi:10.1017/s0950268800048019].
Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses [@doi:10.1016/j.cell.2020.08.025].
An early study (initially released on _medRxiv_ on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% and 57.2% of sera samples from 276 confirmed COVID-19 patients.
They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.
Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].

<!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036].
One study assessed sustained immunity using 245 blood samples from 188 COVID-19 positive patients  [@doi:10.1126/science.abf4063]. 
The samples were collected at various time points between 6 and 240 days post-symptom onset, meaning some patients were assessed longitudinally. 
Of the samples, 43 were collected at least 6 months after symptom onset.
After 1 month, 98% of patients were seropositive for Spike IgG. 
Moreover, Spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
Memory B cells and memory T cells were also measured.
SARS-CoV-2 Spike-specific memory B cell levels steadily rose over the first 120 days following symptom onset. 
RBD-specific memory B cells had been detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @pmid:33443036], and this study confirms these finding and shows that these cells steadily increase over the 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
Likewise, nucleocapsid-specific memory B cells seem to follow a similar pattern [@doi:10.1126/science.abf4063].
SARS-CoV-2 memory CD8+ T cells were also detected in 70% of 169 COVID-19 patients after 1 month [@doi:10.1126/science.abf4063], which is consistent with previous research [@doi:10.1016/j.cell.2020.05.015].
However, SARS-CoV-2 memory CD8+ T cells were slightly decreased (50%) 6 months post-symptom onset. 
In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4+ T cells, of which 42% had > 1% SARS-CoV-2-specific CD4+ T cells.
At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4+ T cells.
Indeed, the spike-specific memory CD4+ T cells abundance over time were similar to the overall SARS-CoV-2-specific CD4+ T cells [@doi:10.1126/science.abf4063].
T cell immunity to SARS-CoV-2 at 6-8 months post-symptom onset has also been confirmed by others [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636].
In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never before exposed to SARS-CoV-2. 
This indicates that there is potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating coronaviruses that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
There is concern that immunity may wane over time, as there several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection
Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481]. 

#### Current Approaches

Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions [@url:https://www.fda.gov/media/136625/download].
Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma (from a blood sample) of patients suspected to have developed the SARS-CoV-2 infection [@url:https://www.fda.gov/media/136625/download].
Such tests illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
The test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
With this particular assay, results can be read within 15-20 minutes [@url:https://www.fda.gov/media/136625/download].
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai, proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
The authors are now working to get the assay into clinical use [@url:https://www.livescience.com/coronavirus-tests-available.html].

#### Limitations of Serological Tests

Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitations due to timing make serological tests far less useful for enabling test-and-trace strategies.

### Possible Alternatives to Current Practices for Identifying Active Cases

Clinical symptoms are too similar to other types of pneumonia to be sufficient as a sole diagnostics criterion.
In addition, as noted above, identifying asymptomatic cases is critical.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% [@doi:10.2807/1560-7917.ES.2020.25.16.2000508].
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice, and for reducing tests needed for patients deemed unlikely to have COVID-19.

X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.

### Strategies and Considerations for Determining Whom to Test

Early in the COVID-19 pandemic, testing was typically limited to individuals considered high risk for developing serious illness [@url:https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html].
This approach often involved limiting testing to people with severe symptoms and people showing mild symptoms that had been in contact with a person who had tested positive.
Individuals who are asymptomatic (i.e., potential spreaders) and individuals who are able to recover at home are therefore often unaware of their status.
However, this method of testing administration misses a high proportion of infections and does not allow for test-and-trace methods to be used.
For instance, a recent study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May, 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
Technological advancements that facilitate widespread, rapid testing will therefore improve the potential to accurately assess the rate of infection and aid in controlling the virus' spread.

### Conclusions

Major advancements have been made in identifying diagnostic approaches.
The development of diagnostic technologies have been rapid, beginning with the release of the SARS-CoV-2 viral genome sequence in January.
As of October 2020, a range of diagnostic tests have become available.
One class of tests uses PCR (RT-PCR or qRT-PCR) to assess the presence of SARS-CoV-2 RNA, while another typically uses ELISA to test for the presence of antibodies to SARS-CoV-2.
The former approach is useful for identifying active infections, while the latter measures hallmarks of the immune response and therefore can detect either active infections or immunity gained from prior infection.
Combining these tests leads to extremely accurate detection of SARS-CoV-2 infection (98.6%), but when used alone, PCR-based tests are recommended before 5.5 days after the onset of the illness and antibody tests after 5.5 days [@doi:10.1001/jama.2020.8259].
Other strategies for testing can also influence the tests' accuracy, such as the use of nasopharyngeal swabs versus BALF [@doi:10.1001/jama.2020.8259], which allow for trade-offs between patient's comfort and test sensitivity.
Additionally, technologies such as digital PCR may allow for scale-up in the throughput of diagnostic testing, facilitating widespread testing.
One major question that remains is whether people who recover from SARS-CoV-2 develop sustained immunity, and over what period this immunity is expected to last.
Some reports have suggested that some patients may develop COVID-19 reinfections (e.g., [@doi:10/ghfgkt]), but the rates of reinfection are currently unknown.
Serologic testing combined with PCR testing will be critical to confirming purported cases of reinfection and to identifying the duration over which immunity is retained and to understanding reinfection risks.
